langchain/experimental/plan_and_execute/agent_executor.py

```diff
@@ -40,6 +40,9 @@ def _call(
             run_manager.on_text(str(plan), verbose=self.verbose)
         for step in plan.steps:
             _new_inputs = {"previous_steps": self.step_container, "current_step": step}
+            if "objective" in self.executor.chain.input_keys:
```

Comment:
isnt this already passed in due to the below line?

`new_inputs = {**_new_inputs, **inputs}`

Reply:
Yep. I thought it might be clearer to explicitly add it, but the information is contained in inputs with the key "input" (and then in `_new_inputs`.

I'll make the change to revert this and change the expected prompt key in `load_agent_executor` to expect "input".

Reply:
@hwchase17 In hindsight, I think this is required but not for the reason I originally thought. I see that the `input_key` can actually be changed in the `PlanAndExecute` class, so i think it's safer to include this logic, and ensure that dictionary passed through includes the expected key. (i.e. if I remove this line and change the template key to `input` but `input_key` changes, we will see this break).

I have pushed a proposed fix :)